Source/Fuse.Controls.Panels/Layouts/WrapLayout.uno

```diff
@@ -135,6 +153,8 @@ namespace Fuse.Layouts
 			var placements = new float4[elements.Count];
 			//minorMaxSize in each major row, assinged per element
 			var minorSizes = new float[elements.Count];
+			// save the available space for each row
+			var majorRest = new Dictionary<float, float>();
```

Comment:
`minorUsed` is a calculated value.

You should only need one `majorRest` per row, shouldn't you?  It's just the left-over major space after sizing each element.